conda/meta.yaml

```diff
@@ -17,6 +22,11 @@ requirements:
     - scippneutron>=24.01
     - scippnexus>=23.12
 
+  {% for package in dependencies %}
```

Comment:
The dependencies above (apart from python310) need to be removed, as this would now be duplicating them all.